install/01_enabling_pg_stat_statements.mdx

```diff
@@ -4,7 +4,7 @@ backlink_href: /docs/install
 backlink_title: 'Installation Guide'
 ---
 
-We use the [pg\_stat\_statements](http://www.postgresql.org/docs/current/static/pgstatstatements.html) extension (officially bundled with PostgreSQL) for tracking which queries get executed in your database.
+We use the [pg_stat_statements](http://www.postgresql.org/docs/current/static/pgstatstatements.html) extension (officially bundled with PostgreSQL) for tracking which queries get executed in your database.
```

Comment:
FYI: I don't think this escaping is necessary, I checked locally and confirmed that it's working fine. Markdown parsers these days are smart enough not to convert this to italic.